src/libraries/Microsoft.PowerFx.Connectors/OpenApiParser.cs

```diff
@@ -170,7 +170,7 @@ private static bool ValidateSupportedOpenApiDocument(OpenApiDocument openApiDocu
                 return false;
             }
 
-            if (!ignoreUnknownExtensions)
+            if (failOnUnknownExtensions)
```

Comment:
>failOnUnknownExtensions [](http://example.com/codeflow?start=16&length=23)

I am not familiar with the entire logic, but want to make sure this flip (removal of `!`) was conscious choice, since I don't see it flip elsewhere L 226, L 257